sdk/monitor/monitor-opentelemetry-exporter/samples-dev/HttpServer.ts

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/**
+ * A function which handles requests and send response.
+ *
+ * @summary a https server
```

Comment:
Same comment about extending this summary: you're recording instances of calls to an API endpoint, so I would add some more to the summary just to get that point.